### PR TITLE
Serial Number Fixes

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
fixes some guns have the wrong or no serial numbers

## Why It's Good For The Game
bugfix good?

## Changelog
:cl:
fix: some guns having wrong or no serial numbers
:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
why the hell do all my PRs keep being made as drafts

**Comment by thedarkcrusader** [comment]
> why the hell do all my PRs keep being made as drafts

Dont click the draft button

File: code/modules/projectiles/guns/projectile/automatic/slaught_o_matic.dm [hunk 1]
```diff
@@ -14,6 +14,7 @@
 	load_method = MAGAZINE
 	mag_well = MAG_WELL_SMG
 	auto_eject = FALSE
+	serial_type = "FS"
```
Comment (Firefox13) [inline code/modules/projectiles/guns/projectile/automatic/slaught_o_matic.dm:17]: I'm pretty sure idea of these guns is to be untraceable but we don't have "There's no serial number whatsoever" so ehhhhh

**Review by Firefox13 — APPROVED** [review]

**Review by TheShown911 — APPROVED** [review]
Code is good.

**Pull request merged**